parrot/src/syscall_parrot.tbl

```diff
@@ -14,4 +14,5 @@
 1009 parrot parrot_search sys_parrot_search
 1010 parrot parrot_debug sys_parrot_debug
 1011 parrot parrot_mount sys_parrot_mount
-1012 parrot parrot_unmount sys_parrot_unmount
\ No newline at end of file
+1012 parrot parrot_unmount sys_parrot_unmount
+1013 parrot parrot_fcopyfile sys_parrot_fcopyfile
```

Comment:
Ya @trshaffer, this 1013 syscall isn't necessary. [If you wanted to add a new syscall, you need to implement it in `pfs_dispatch*.cc`.] Go ahead and remove it.